Makefile

```diff
@@ -6,6 +6,9 @@ SHELL = bash
 init:
 	find .git/hooks -type l -exec rm {} \;
 	find .githooks -type f -exec ln -sf ../../{} .git/hooks/ \;
+	mkdir output/
```

Comment:
after we merge to master do we all need to call `make init` again?

Reply:
Correct

Reply:
After this merges, everyone should immediately pull. This PR changes quite a lot.